src/app/app.component.ts

```diff
@@ -26,4 +38,37 @@ export class AppComponent implements OnInit {
       }
     });
   }
+
+  private setMetasSubject(): void {
```

Comment:
Please rename it, setter should has some argument

Reply:
ok